onnx/helper.py

```diff
@@ -1098,3 +1095,121 @@ def make_training_info(
             binding.value = v
 
     return training_info
+
+
+# Following functions are used for mapping
+def tensor_dtype_to_np_type(tensor_dtype: int) -> np.dtype:
+    """
+    Convert a TensorProto's data_type to corresponding numpy dtype. It can be used while making tensor.
+
+    :param tensor_dtype: TensorProto's data_type
+    :return: numpy's data_type
+    """
+    return mapping.TENSOR_TYPE_MAP[int(tensor_dtype)].np_type
+
+
+def tensor_dtype_to_storage_tensor_type(tensor_dtype: int) -> int:
```

Comment:
why dtype sometimes and only type some other types?

Reply:
Good catch. Updated by always using dtype.